modules/test/playwright/tests/FDS/FDSActions.spec.ts

```diff
@@ -0,0 +1,90 @@
+import {expect, mergeTests} from '@playwright/test';
+
+import {FDSPagesTest} from '../../fixtures/FDSPages.fixture';
+
+export const test = mergeTests(
+	FDSPagesTest
+);
+
+test.describe('LPS-186875 Create generic creation button actions', () => {
+
+    test.afterAll(async () => {
+        console.log('Done with tests...time to clean the database (still in progress, see https://playwright.dev/docs/test-retries#reuse-single-page-between-tests)');
+    });
+
+    test('Data Set Test is created', async ({page, _FDSPage}) => {
+        await _FDSPage.goto();
+
+        await expect(
+            page.getByRole('link', { name: /Data Set Test/ }).first()
+        ).toBeVisible();
+    });
+
+    test('View Test is created', async ({page, _FDSViewPage}) => {
+        await _FDSViewPage.createTestDataSetView();
+
+        await expect( 
+            page.getByRole('heading', {name: /New Data Set View/})
+		).toBeVisible();
+    });
+
+    test('There are no item actions created in the Action tab', async ({page, _FDSActionsPage}) => {
+        await _FDSActionsPage.goto();
+   
+        await expect(page.getByRole('tabpanel').getByText(/No actions were created./)).toBeVisible();
+    });
+
+    test('The "New Creation Action" button is present', async ({page, _FDSActionsPage}) => {
+        await _FDSActionsPage.goto();
+
+        await expect(
+            page.getByRole('button', { name: /New Item Action/ })
+        ).toBeVisible();
+    });
+ 
+    test('A new Link action is created', async ({page, _FDSActionsPage}) => {
+        await _FDSActionsPage.goto();
```

Comment:
this one as well as modal actions and side panel actions, should not belong to `LPS-186875 Create generic creation button actions`. It's good to have them around as we'll need them for sure, however, they should go to specific sections where those type of actions are tested

Reply:
Yeah! That's what I tried to mean here : https://github.com/liferay-frontend/liferay-portal/pull/3924#discussion_r1448678255